Makefile

```diff
@@ -373,6 +376,12 @@ PACK = -if [ -z $(4) ]; then \
 	echo "Installed-Size: $$SIZE" >> $(BUILD_DIST)/$(1)/DEBIAN/control; \
 	find $(BUILD_DIST)/$(1) -name '.DS_Store' -type f -delete; \
 	$(FAKEROOT) $(DPKG_DEB) -b $(BUILD_DIST)/$(1) $(BUILD_DIST)/$$(grep Package: $(BUILD_DIST)/$(1)/DEBIAN/control | cut -f2 -d ' ')_$($(2))_$$(grep Architecture: $(BUILD_DIST)/$(1)/DEBIAN/control | cut -f2 -d ' ').deb
+	if [ "$(TO_INSTALL)" = "1" ]; then \
```

Comment:
Instead of doing this, why not have it pack all the stuff for the package you want to install in a different directory (build_dist/**/(package)?), then rsync the entire directory over to /tmp, and install every package in that directory.